src/plone/exportimport/importers/__init__.py

```diff
@@ -37,7 +38,7 @@ def __init__(self, site):
 
     def all_importers(self) -> List[BaseImporter]:
         """Return all importers."""
-        importers = {}
+        importers = OrderedDict()
```

Comment:
Why use the OrderedDict here? Since Python 3.6 all dictionaries are ordered by default. Am I missing something?

Reply:
Oops, no, my fault.
Corrected.